src/Controller/LectureController.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: grantas
+ * Date: 18.3.30
+ * Time: 13.59
+ */
+
+namespace App\Controller;
+use App\Entity\Lecture;
+use App\Form\LectureType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+
+class LectureController extends Controller
+{
+
+    /**
+     * @Route("api/lectures", name="api_lecture_create", methods="POST")
+     */
+    public function setLecture(Request $request)
```

Comment:
Kodėl čia set o route name'as - create? :) čia nieko esminio, tiesiog be consistent